### PR TITLE
Rollbar environment

### PR DESCRIPTION
**Pull request by gudmundur**
See discussion here: https://github.com/interagent/pliny/pull/160.

@rwz what do you think?

### CODE REVIEW

**Comment by rwz** [comment]
:+1:

File: lib/template/config/initializers/rollbar.rb [hunk 1]
```diff
@@ -1,6 +1,7 @@
 Rollbar.configure do |config|
   config.enabled = ENV.has_key?('ROLLBAR_ACCESS_TOKEN')
   config.access_token = ENV["ROLLBAR_ACCESS_TOKEN"]
+  config.environment = ENV['ROLLBAR_ENVIRONMENT']
```
Comment (soulim) [inline lib/template/config/initializers/rollbar.rb]: Would it make sense to add `ROLLBAR_ENVIRONMENT` to [`pliny/lib/template/.env.sample.erb`](https://github.com/interagent/pliny/blob/fdf30c530367d07bae2064527019bfd38edbf7dc/lib/template/.env.sample.erb)?

```
ROLLBAR_ENVIRONMENT=development
```

**Comment by soulim** [comment]
I know it's just cosmetics, but would it make sense to use `_ENV` suffix instead of `_ENVIRONMENT`? Then variable name would be consistent with `RACK_ENV`, `PLINY_ENV`, and etc.

File: lib/template/config/initializers/rollbar.rb (hunk 1 above)
Comment (gudmundur) [inline lib/template/config/initializers/rollbar.rb]: I intentionally left this out, as `ROLLBAR_ACCESS_TOKEN` isn't specified there. Which causes it to default to being disabled.

**Comment by gudmundur** [comment]
@soulim Excellent point. I'll get it updated.

File: lib/template/config/initializers/rollbar.rb (hunk 1 above)
Comment (soulim) [inline lib/template/config/initializers/rollbar.rb]: Yeah, it makes sense. Thanks for explanation! :+1:

I wasn't sure how Rollbar gem acts when `environment` is `nil`, but seems like it [handles everything pretty well](https://github.com/rollbar/rollbar-gem/blob/d06b18d2a98d8effdde605355ebe14add7ae5d3d/lib/rollbar.rb#L368-L369) :smile:

**Pull request merged**